EKF/terrain_estimator.cpp

```diff
@@ -156,3 +169,27 @@ void Ekf::get_hagl_innov_var(float *hagl_innov_var)
 {
 	memcpy(hagl_innov_var, &_hagl_innov_var, sizeof(_hagl_innov_var));
 }
+
+// check that the range finder data is continuous
+void Ekf::checkRangeDataContinuity()
+{
+	// update range data continuous flag (2Hz ie 500 ms)
+	/* Timing in micro seconds */
+	static hrt_abstime t = 0;
+	static hrt_abstime t_prev = 0;
+	static float dt = 0.0f;
+	t = hrt_absolute_time();
```

Comment:
@priseborough  I think it would be good if we can avoid using calls to the hrt timer in order to avoid dependencies.
I'll look how we can solve this.